### PR TITLE
E_CLASSROOM-215-FE/BE-Profile-Edit-Function - first commit

### PR DESCRIPTION
**Pull request by CaneteJoash**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-215
### Definition of Done
- [x] User edit profile name

### Commands to Run
N/A

#### Main note
- FE LINK: https://github.com/framgia/sph-classroom-els-fe/pull/76
#### Pages Affected
- `http://localhost:82/api/v1/profileEdit`

### Scenarios/ Test Cases
- [x] User edit profile name
#### User Interface
N/A
#### User Experience 
N/A
#### Functionality
N/A
### Screenshots
image

### CODE REVIEW

**Comment by catampongan** [comment]
## Approved PR #44 
- [x] Able to change name
- [x] Display validation when password incorrect

------

name-change

------

- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

File: app/Http/Requests/Auth/ChangeNameEmailRequest.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        ];
+    }
+
+    public function messages() 
+    {
+      return [
+        'email.ends_with' => 'Your email is not valid',
+      ];
+    }
+}
```
Comment (karlolee50) [inline app/Http/Requests/Auth/ChangeNameEmailRequest.php:38]: Is it possible to reuse the existing rules for email checking?

File: app/Http/Requests/Auth/UploadImageRequest.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'image' => 'required|image|mimes:jpeg,png,jpg,gif,svg|max:1000',
```
Comment (karlolee50) [inline app/Http/Requests/Auth/UploadImageRequest.php]:
```suggestion
            'image' => 'required|image|mimes:jpeg,png,jpg,gif,svg|max:1024',
```

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Is it possible to delete the image files? These will be included in the develop branch if we merge this PR

image

**Review by karlolee50 — APPROVED** [review]
Review time: 15 minutes
Comment/s: 2

**Pull request merged**